toolchain/build-toolchain.sh

```diff
@@ -1,69 +1,80 @@
-#!/bin/bash
+#!/usr/bin/env sh
 
-MAKE_OPTIONS="-j1"
-
-function download() { # $1=url, $2=file
-	if [ ! -f $2 ]; then
-		curl "$1$2" > "$2"
-	fi
-
-	if [ ! -d "${2%.tar.xz}" ]; then
-		tar xf "$2"
-	fi
+download() {
+	wget "$1" -O "$2"
+	tar xvf "$2"
```

Comment:
That little `v` is pretty spammy, maybe a single print like "Extracting $2" ?

Reply:
That's true. My fix is this.
```suggestion
log() {
	printf "\033[33m->\033[m %s\n" "$*"
}

download() {
	log "Downloading $2"
	wget "$1" -O "$2"
	log "Inflating $2"
	tar xf "$2"
}
```
Any thoughts?

Reply:
Looks great!